core/src/test/java/com/linecorp/armeria/server/encoding/DecoderLengthLimitTest.java

```diff
@@ -60,10 +60,10 @@ void decodedDataShouldNotExceedLengthLimit(HttpEncodingType encodingType) throws
         assertThatThrownBy(() -> {
             strictStreamDecoder.decode(httpData);
         }).isInstanceOf(ContentTooLargeException.class)
-          .satisfies(cause -> {
-              final ContentTooLargeException tooLargeException = (ContentTooLargeException) cause;
-              assertThat(tooLargeException.maxContentLength()).isEqualTo(9999);
-          });
+                .satisfies(cause -> {
```

Comment:
Could you also revert the format changes in this file? 😉

Reply:
apologies, will fix that.

Reply:
@minwoox fixed the formatting issues. please let me know if there is anything else missing. thanks!